docs(policy): clean up targetRef explanation (#1567)
kumahq/kuma-website: app/_src/policies/targetref.md

## What do `targetRef` policies look like?

There are two parts to a policy:

1. The metadata
2. The spec

### Metadata

> Metadata identifies the policies by its `name`, `type` and what `mesh` it is part of:

Reviewer: ```suggestion
Metadata identifies the policies by its `name`, `type` and what `mesh` it's part of:
```
